docs/docs/reference/other-new-features/indentation.md

```diff
@@ -105,6 +108,24 @@ if x < 0 then
 Indentation tokens are only inserted in regions where newline statement separators are also inferred:
 at the top-level, inside braces `{...}`, but not inside parentheses `(...)`, patterns or types.
 
+**Note:** The rules for leading infix operators above are there to make sure that
+```scala
+  one
+  + two.match
+      case 1 => b
+      case 2 => c
+  + three
+```
+is parsed as `one + (two.match ...) + three`. Also, that
+```scala
+if x then
+    a
+  + b
+  + c
+else d
+```
+is parsed as `if x then a + b + c else d`.
```

Comment:
The following code compiles previously:

```Scala
class Test:
   if
     5 < 6
   + 3
   then
    println("hello")
```

But I guess it's fine to reject it after the change.

Reply:
I think it's OK, in particular since there is no legacy code, since leading infix operators are a Scala 3 thing.